src/javascript/Edit/copyLanguage/copyLanguage.utils.js

```diff
@@ -0,0 +1,12 @@
+
+/**
+ * This function get the full name of a language by his id
+ *
+ * @param languages list of available languages
+ * @param id language id to get
+ */
+export function getFullLanguageName(languages, id) {
```

Comment:
please add test to that utility class, thank you

Reply:
Tests added